sfxclient/README.md

```diff
@@ -16,12 +16,27 @@ libraries.
 The simplest way to send metrics to SignalFx is with HTTPSink. The only
 struct parameter that needs to be configured is AuthToken. To make it easier to
 create common Datapoint objects, wrappers exist for Gauge and Cumulative. An
-example of sending a hello world metric would look like this:
+example of sending a hello world metric would look like shown below:
+
+#### Configuring your endpoints
+
+If no endpoints are set manually, this library uses the `us0` realm by default.
```

Comment:
This info has to be above the previous paragraph, if possible, because it says "shown below:" which is the code sample. If that won't work, just remove the colon.